README.md

```diff
@@ -9,9 +9,9 @@ This repository exists to host the documentation for the Egeria project across i
 
 ## Contributing
 
-This project welcomes contributors from any organization or background, provided they are willing to follow the simple processes outlined below, as well as adhere to the [Code of Conduct](https://github.com/odpi/specs/wiki/ODPi-Code-of-Conduct). Review the [community guide](https://odpi.github.io/guides/community) to find out more.
+This project welcomes contributors from any organization or background, provided they are willing to follow the simple processes outlined below, as well as adhere to the [Code of Conduct](https://github.com/odpi/specs/wiki/ODPi-Code-of-Conduct). Review the [community guide](https://egeria-project.org/guides/community) to find out more.
 
-To review how to specifically contribute to the documentation, review our [documentation guide](https://odpi.github.io/guides/documentation/guide).
+To review how to specifically contribute to the documentation, review our [documentation guide](https://egeria-project.org/guides/documentation/guide).
```

Comment:
Keeping with the internal link idea- this should also be an internal link

Reply:
See above comment